cmd/ebrelayer/txs/signature.go

```diff
@@ -21,13 +22,15 @@ func LoadPrivateKey() (key *ecdsa.PrivateKey, err error) {
 	// Private key for validator's Ethereum address must be set as an environment variable
 	rawPrivateKey := os.Getenv("ETHEREUM_PRIVATE_KEY")
 	if strings.TrimSpace(rawPrivateKey) == "" {
-		log.Fatal("Error loading ETHEREUM_PRIVATE_KEY from .env file")
+		log.Println("Error loading ETHEREUM_PRIVATE_KEY from .env file")
```

Comment:
Can we just change this line too:

```
 log.Println("Error loading ETHEREUM_PRIVATE_KEY") 
```
As we wont be using `.env` files in all deployments

Reply:
Yes, .env file just for local machine, will remove it.